### PR TITLE
Feature/KITOSUDV-3213

### PR DESCRIPTION
**Pull request by a-naskret**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [ ] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [ ] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html [hunk 1]
```diff
@@ -76,7 +76,7 @@ <h4 id="systemUsageHeaderName">{{systemUsageName}}</h4>
                            data-ng-disabled="true"
                            value="{{usage.objectOwnerFullName}}">
                 </div>
-                <div class="form-group">
+                <div class="form-group" ng-if="showDateFields">
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html:79]: not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data

File: Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html [hunk 2]
```diff
@@ -88,7 +88,7 @@ <h4 id="systemUsageHeaderName">{{systemUsageName}}</h4>
                            ng-blur="patchDate('concluded', usage.concluded)"
                            data-field="concluded">
                 </div>
-                <div class="form-group">
+                <div class="form-group" ng-if="showDateFields">
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html:91]: not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data

File: Presentation.Web/app/models/ui-customization/config/blueprints/it-system-usage-ui-customization-config-blueprint.ts [hunk 3]
```diff
@@ -8,7 +8,16 @@
             frontPage: {
                 text: "Systemforside",
                 readOnly: true,
-                helpText: Configs.helpTexts.cannotChangeTab
+                helpText: Configs.helpTexts.cannotChangeTab,
+                children: {
+                    dateFields: {
```
Comment (mrjsawdk) [inline Presentation.Web/app/models/ui-customization/config/blueprints/it-system-usage-ui-customization-config-blueprint.ts:13]: not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data

File: Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts [hunk 4]
```diff
@@ -28,6 +34,9 @@
             $scope.isValidUrl = (url: string) => Kitos.Utility.Validation.isValidExternalReference(url);
             const saveUrlWithOrgId = $scope.autoSaveUrl + "?organizationId=" + user.currentOrganizationId;
 
+            $scope.showDateFields = uiState.isBluePrintNodeAvailable(blueprint.children.frontPage.children.dateFields);
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts:37]: not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data

File: Presentation.Web/app/components/it-system/it-system-overview.controller.ts [hunk 5]
```diff
@@ -259,7 +259,7 @@
                     .withFixedValueRange(createActiveRange(), false)
                     .withRendering(dataItem => Helpers.RenderFieldsHelper.renderActiveNotActive(dataItem.ActiveAccordingToValidityPeriod))
                     .withContentAlignment(Utility.KendoGrid.KendoColumnAlignment.Center)
-                    .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.frontPage)))
+                    .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.frontPage.children.dateFields)))
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/it-system/it-system-overview.controller.ts:262]: what about the individual date fields... would like their columns to be hidden as well

Comment (mrjsawdk) [reply Presentation.Web/app/components/it-system/it-system-overview.controller.ts:262]: II could only find a column for "concluded" (ibrugtagningsdato) but that should be covered by this condition as well

**Review by mrjsawdk — COMMENTED** [review]
@a-naskret Some adjustments in terms of naming, but other than that it looks correct to me..

Perhaps also merge master in and re-check green on integration.

Then we will do a final quick-review

File: Presentation.Web/app/components/it-system/it-system-overview.controller.ts (hunk 5 above)
Comment (a-naskret) [reply Presentation.Web/app/components/it-system/it-system-overview.controller.ts:262]: it's covered 
image

Comment (mrjsawdk) [reply Presentation.Web/app/components/it-system/it-system-overview.controller.ts:262]: @a-naskret Excellent

**Pull request closed without merging**